terraso_backend/config/settings.py

```diff
@@ -211,7 +211,11 @@
     "TESTING_ENDPOINT": "/graphql/",
 }
 
-WEB_CLIENT_URL = config("WEB_CLIENT_ENDPOINT", default="")
+
+WEB_CLIENT_DOMAIN = config("WEB_CLIENT_DOMAIN", default="")
+WEB_CLIENT_URL = f"https://{WEB_CLIENT_DOMAIN}"
```

Comment:
Please support custom protocol and port (so local development works) per the example I sent you.

Reply:
done